packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -661,7 +661,7 @@ export const makeLegacySendBatchFn =
 	(batch: IBatch) => {
 		// Default to negative one to match Container.submitBatch behavior
 		let clientSequenceNumber: number = -1;
-		for (const message of batch.content) {
+		for (const message of batch.messages) {
```

Comment:
Is this backwards compatible? Do we need to worry about extremely old batch messages?

Reply:
If this is a serialized format I will revert it. I assumed it was just a runtime type.  It's not tagged or anything.

Reply:
Looking at the submit codepath, this type is not directly used for serialization.  Its members get passed around individually over several function calls, final format is `IDocumentMessage`.